h2o-docs/src/product/performance-and-prediction.rst

```diff
@@ -699,6 +699,28 @@ For classification problems, when running ``h2o.predict()`` or ``.predict()``, t
 - If you train a model with train data and set the ``nfold`` parameter, the Max F1 threshold from the training data model metrics is used.
 - If you train a model with the train data and validation data and also set the ``nfold parameter``, the Max F1 threshold from the validation data model metrics is used.
 
+Predict Feature Frequency
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use the ``feature_frequencies`` function to retrieve the number of times a feature was used on a predication path in a tree model. This option is only available in GBM, DRF, and IF.
+
+Using the previous example, run the following to predict the leaf node assignments:
```

Comment:
"Using the previous example, run the following to the find frequency of each feature in the predication path of the model:"